htdocs/wp-config.php

```diff
@@ -105,9 +104,6 @@
   define('WP_DEBUG_DISPLAY', true);
   define('WP_DEBUG_LOG', '/data/log/php-error.log');
   define('SCRIPT_DEBUG', true);
-
-  // Disable wp-content/object-cache.php from being active during development
-  define('WP_REDIS_DISABLED', true);
```

Comment:
Could commenting this be more suitable? That way if developer faces problems, it would be easy for them to enable redis-free dev-environment.

Of course, the comment above should be altered accordingly.

Reply:
The developer can just delete the object-cache.php file or rename it o .off, like we have on many sites. The proper solution would to write on seravo.com/docs/ more extensive story on Redis and how it works, how to debug etc.

Reply:
Ah yeah that's true! Unrelevant for this PR then. Resolving